packer/scylla_install_image

```diff
@@ -115,14 +115,17 @@ if __name__ == '__main__':
         setup_opt = '--ntp-domain amazon'
         sysconfig_opt = ''
         swap_opt = '--swap-directory /'
+        kernel_opt = ''
     elif args.target_cloud == 'gce':
         setup_opt = ''
         sysconfig_opt = '--disable-writeback-cache'
         swap_opt = '--swap-directory /'
+        kernel_opt = ' scsi_mod.use_blk_mq=Y'
```

Comment:
Based on the image created during the verification you did, i don't see this option available @syuu1228